sdk/identity/identity/src/msal/browserFlows/browserCommon.ts

```diff
@@ -143,6 +143,8 @@ export abstract class MsalBrowser extends MsalBaseUtilities implements MsalBrows
       }
       if (options?.disableAutomaticAuthentication) {
         throw new AuthenticationRequired(
```

Comment:
That's a great question. We shipped `CredentialUnavailable` with our 1.x.x GA ([API link](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/identity_1.2.5/sdk/identity/identity/review/identity.api.md)) however, we could change it since this is 2.x.x. Let me ping Brian, @bterlson

Reply:
How prevalent would usages of these types be? Do you have any samples showing the usage of "instanceof" checks for error handling, for example? Hopefully everyone is using `.name`. If there's little risk of someone having a reason to use these types exactly I'm happy to append Error for 2.0.

Reply:
@bterlson on 2.0.0 we're always using `.name` to check for these types. `CredentialUnavailable` is used on all credentials, specially to make the ChainedTokenCredential move ahead. `AuthenticationRequired` is mostly internal, it tells us that a silent authentication failed, but if users disable automatic authentication (new option on constructors), then they will get this error if silent authentication fails, and they can decide when to manually authenticate (with `authenticate()`). Does that help?